<launch>
	<arg name="rviz" default="true" />
  <remap from="/velodyne_cloud_2" to="/kitti_player/hdl64e"/>

	<node pkg="generalp" type="NDTmatchingPCL" name="NDTmatchingPCL" output="screen" respawn="true">
		<param name="voxel_size" type="double" value="0.20"/>
		<param name="Grid_res" type="double" value="1.00"/>
		<param name="epsilon" type="double" value="0.0065"/>
		<param name="stepSize" type="double" value="2.24"/>
	</node>

  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find ndt_mod)/rviz_cfg/loam_velodyne.rviz" />
  </group>
<param name="use_sim_time" value="false" />

</launch>
